mod/settings.py

```diff
@@ -78,6 +78,10 @@
 CONTROLCHAIN_HTTP_ADDRESS = os.environ.pop('MOD_CONTROLCHAIN_HTTP_ADDRESS',
                                            "http://download.moddevices.com/releases/cc-firmware/v1")
 
+MIDI_BEAT_CLOCK_SENDER_URI = "http://gareus.org/oss/lv2/mclk"
```

Comment:
this is the regular plugin, you need to use the custom hardcoded "urn:mod:mclk" URI instead.